lib/merchant.rb

```diff
@@ -1,32 +1,14 @@
 class Merchant
   attr_reader :id,
               :name
-  def initialize(id, name)
-    @id = id
+  def initialize(id, name) #parent)
+    @id = id.to_i
     @name = name
+    #@parent = parent
   end
 
-
-  def highest_merchant_id_plus_one
-    highest = merchants.max do |merchant|
-      merchant[:id]
-    end
-    highest + 1
-  end
-
-  # def create(args)
-  #   @merchants << merchant.new({:id => highest_merchant_id_plus_one,
-  #                 :args[:name] => ?)
-  # end
-
-  def update(id, attributes)
-
+  def update(name)
```

Comment:
Good call to have this method on the merchant class itself. A merchant should be able to update its name!

Reply:
Yeah, i didn't think it made sense to build that into the merchant_repo class. I'm still a little on the fence of whether or not it makes sense to have the update and delete methods in the repo class. Do you think that was a smart move or should we move them to merchant class?

Reply:
I think that it's a good idea to have it in the merchant class. If we need to change it we can always make the move! 👍 Great work dev!